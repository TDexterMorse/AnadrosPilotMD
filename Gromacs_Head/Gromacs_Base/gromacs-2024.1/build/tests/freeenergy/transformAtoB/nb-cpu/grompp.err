                      :-) GROMACS - gmx grompp, 2024.1 (-:

Executable:   /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/bin/gmx
Data prefix:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1 (source tree)
Working dir:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/freeenergy/transformAtoB/nb-cpu
Command line:
  gmx grompp -f /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/transformAtoB/grompp.mdp -c /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/transformAtoB/conf -r /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/transformAtoB/conf -p /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/transformAtoB/topol -maxwarn 10

Ignoring obsolete mdp entry 'rlistlong'
Ignoring obsolete mdp entry 'nstcalclr'
Ignoring obsolete mdp entry 'gb-algorithm'
Ignoring obsolete mdp entry 'nstgbradii'
Ignoring obsolete mdp entry 'rgbradii'
Ignoring obsolete mdp entry 'gb-epsilon-solvent'
Ignoring obsolete mdp entry 'gb-saltconc'
Ignoring obsolete mdp entry 'gb-obc-alpha'
Ignoring obsolete mdp entry 'gb-obc-beta'
Ignoring obsolete mdp entry 'gb-obc-gamma'
Ignoring obsolete mdp entry 'gb-dielectric-offset'
Ignoring obsolete mdp entry 'sa-algorithm'
Ignoring obsolete mdp entry 'sa-surface-tension'
Ignoring obsolete mdp entry 'ns_type'

NOTE 1 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/transformAtoB/grompp.mdp]:
  Replacing vdwtype=Switch by the equivalent combination of vdwtype=Cut-off
  and vdw_modifier=Potential-switch


NOTE 2 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/transformAtoB/grompp.mdp]:
  You have set rlist larger than the interaction cut-off, but you also have
  verlet-buffer-tolerance > 0. Will set rlist using
  verlet-buffer-tolerance.

Generating 1-4 interactions: fudge = 0.5
Number of degrees of freedom in T-Coupling group System is 5394.00

WARNING 1 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/transformAtoB/grompp.mdp]:
  You are not using center of mass motion removal (mdp option comm-mode),
  numerical rounding errors can lead to build up of kinetic energy of the
  center of mass


There were 2 NOTEs

There was 1 WARNING

Thanx for Using GROMACS - Have a Nice Day

