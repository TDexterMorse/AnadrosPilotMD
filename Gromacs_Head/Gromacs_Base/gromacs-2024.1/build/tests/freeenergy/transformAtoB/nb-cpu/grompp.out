Generated 210 of the 210 non-bonded parameter combinations

Generated 210 of the 210 1-4 parameter combinations

Excluding 3 bonded neighbours molecule type 'Trp_Side_Chain'

turning H bonds into constraints...

Excluding 1 bonded neighbours molecule type 'SOL'

turning H bonds into constraints...

Velocities were taken from a Maxwell distribution at 298 K
Analysing residue names:
There are:     1      Other residues
There are:   891      Water residues
Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...

The largest distance between non-perturbed excluded atoms is 0.153 nm between atom 2451 and 2452

Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 300 K

Calculated rlist for 1x1 atom pair-list as 0.918 nm, buffer size 0.018 nm

Set rlist, assuming 4x4 atom pair-list, to 0.903 nm, buffer size 0.003 nm

Note that mdrun will redetermine rlist based on the actual pair-list setup
Calculating fourier grid dimensions for X Y Z
Using a fourier grid of 32x32x32, spacing 0.094 0.094 0.094

Estimate for the relative computational load of the PME mesh part: 0.66

This run will generate roughly 1 Mb of data
